x-pack/plugins/alerting/server/routes/rule/apis/find/find_rules_route.ts

```diff
@@ -90,6 +104,86 @@ export const findRulesRoute = (
       })
     )
   );
+  if (path === INTERNAL_ALERTING_API_FIND_RULES_PATH) {
```

Comment:
I am a bit confused by this change, there is already a `findInternalRulesRoute` in `x-pack/plugins/alerting/server/routes/rule/apis/find/find_internal_rules_route.ts` does it not provide what you need?

Also, this function creates a `router.get` **and** a `router.post` when the path is `INTERNAL_ALERTING_API_FIND_RULES_PATH` is this intended? I'm still navigating the changes looking for where the api is used.

Reply:
Hi 👋 

I am a bit confused as to what is happening here myself. I did not write this code, something strange must've happened with a merge. I'll revert these changes and see how things go. Sorry for this noise. If we end up needing to touch this code I will reach back out.